tools/underlay.repos

```diff
@@ -35,3 +35,8 @@ repositories:
     type: git
     url:  https://github.com/ros-navigation/nav2_minimal_turtlebot_simulation.git
     version: 091b5ff12436890a54de6325df3573ae110e912b
+  ros/common_interfaces:
```

Comment:
just for CI testing. Do you know when we can remove this? Are rolling binaries released nightly?

Reply:
No, they are released monthlyish like other ROS distributions - it'll take a few weeks before we can remove it